pseudomonas_analysis/nbconverted/viz_expression_data.py

```diff
@@ -379,201 +121,11 @@ def make_volcano_plot_simulated_notransform(
-normalized_all_data_UMAPencoded_df['sample group'] = normalized_all_data['sample group']
-
-# Plot
-fig = pn.ggplot(normalized_all_data_UMAPencoded_df, pn.aes(x='1', y='2'))
-fig += pn.geom_point(pn.aes(color='sample group'), alpha=0.4)
-fig += pn.labs(x ='UMAP 1',
-            y = 'UMAP 2',
-            title = 'Gene expression data in gene space')
-fig += pn.theme_bw()
-fig += pn.theme(
-    legend_title_align = "center",
-    plot_background=pn.element_rect(fill='white'),
-    legend_key=pn.element_rect(fill='white', colour='white'), 
-    legend_title=pn.element_text(family='sans-serif', size=15),
-    legend_text=pn.element_text(family='sans-serif', size=12),
-    plot_title=pn.element_text(family='sans-serif', size=15),
-    axis_text=pn.element_text(family='sans-serif', size=12),
-    axis_title=pn.element_text(family='sans-serif', size=15)
-    )
-fig += pn.scale_color_manual(['#bdbdbd', 'red', 'blue'])
-fig += pn.guides(colour=pn.guide_legend(override_aes={'alpha': 1}))
-
-fig += pn.scales.xlim(9,10)
-print(fig)
-
-
-# Based on a UMAP of the normalized gene expression data, it looks like there isn't a clear separation between WT and mutant samples, though there are only 2 samples per group so this type of clustering observation is limited.
-# 
 # **Takeaway:**
 # 
-# In trying to understand why there are these flat-tops to some of the volcano plots and why some volcano plots are completely flat, we found:
-# 1. This behavior is _not_ a result of how we are plotting in python (there was some speculation about there being an issue with the numpy library used)
-# 2. The latent space shifting we're doing seems to roughly preserve differences between groups (as seen in [this notebook](https://github.com/greenelab/simulate-expression-compendia/blob/master/Pseudo_experiments/create_heatmap.ipynb) where the structure of the samples is preserved but there is a different set of related genes that are DE. More information can be found in Figure 3D in [this paper](https://academic.oup.com/gigascience/article/9/11/giaa117/5952607)), but this signal can be muddled/noisy depending on where the experiment was shifted to (i.e. the representation that is found in that location can cause the experiment to have a more compressed difference between groups) as seen in the heatmaps. The heatmap of the two simulation experiments shows that some experiments have a more noisey distinction between groups (WT vs mutant) whereas the other simulation experiment has a more distinct difference where the within grouping is cleaner. This definitely points to the need to understand how this simulation process is working and how biology is represented in the latent space. This will definitely be a project for the future. For now we at least have an explanation for why we are observing these shapes in the volcano plots
+# These highlighted volcano plots demonstrate that the generic gense are found to be highly DE (based on logFC) across all simulated experiments. 
```

Comment:
So, another way of stating this is that the generic genes tend to be toward the tails of your plot? If that's what you're saying I agree (and I guess that makes sense based on your definition of generic gene).

Reply:
Yes that is what I'm saying. This was suggested by my collaborator as a nice visual for people to understand that these generic genes are consistently found in many experiments